clients/admin-ui/src/features/user-management/AssignSystemsTable.tsx

```diff
@@ -66,9 +56,14 @@ export const AssignSystemsDeleteTable = () => {
                 aria-label="Unassign system from user"
                 icon={<TrashCanSolidIcon />}
                 size="xs"
-                onClick={() => handleDelete(system)}
+                onClick={removeSystemModal.onOpen}
                 data-testid="unassign-btn"
               />
+              <RemoveSystemModal
```

Comment:
I'd prefer if we're able to use the `ConfirmationModal` in `common` just so that if we have to update our confirmation modals we can do it all in once place. but let me know if there's a good reason to keep this one as a separate component 👍

Reply:
Sure thing, let me see about replacing this with our `ConfirmationModal`